packages-exp/auth-compat-exp/src/recaptcha_verifier.ts

```diff
@@ -33,6 +33,10 @@ export class RecaptchaVerifier
     impl.assertFn(app.options?.apiKey, impl.AuthErrorCode.INVALID_API_KEY, {
       appName: app.name
     });
-    super(container, parameters, (app.auth!() as unknown) as externs.Auth);
+    super(
+      container,
+      parameters as any,
```

Comment:
I cast it to `any` to workaround the TS error - 
```
Argument of type 'object | null | undefined' is not assignable to parameter of type 'Parameters'.
  Type 'undefined' is not assignable to type 'Parameters'.
```
The error seems legit. Is the casting safe here?

Reply:
Yeah the typing in the parent class is wrong. I'll need to go in and fix it there